modules/ROOT/pages/understanding-enterprise-integration-patterns-using-mule.adoc

```diff
@@ -0,0 +1,92 @@
+= Understanding Enterprise Integration Patterns Using Mule
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Enterprise Integration Patterns are accepted solutions to recurring problems within a given context. The patterns provide a framework for designing, building messaging and integration systems, as well as a common language for teams to use when architecting solutions.
+
+Mule supports most of these patterns shown in the Enterprise Integration Patterns book written by Gregor Hohpe and Bobby Woolf.
+
+By implementing these patterns, Mule reduces the effort required when building integrations. You can use these well-known patters to design your solution and then simply configure and use these same patters in Mule.
+
+
+== Mapping Enterprise Integration Patterns into Mule Objects
+
+There is a list of patterns that can be mapped directly with Mule objects. Review the following list of Enterprise Integration Patterns mapped to Mule objects:
+
```

Comment:
Suggest replacing the Level-2 heading above and this text with the replacements above.